app/src/app/constants/configuration.ts

```diff
@@ -1,14 +1,1939 @@
+        ["linear"],
+        ["zoom"],
+        0,
+        0,
+        8,
+        4,
+        10,
+        8,
+        12,
+        6,
+        22,
+        2,
+      ],
+      "text-anchor": ["step", ["zoom"], "left", 8, "center"],
+      "text-radial-offset": 0.4,
+    },
+    paint: {
+      "text-color": "#5c5c5c",
+      "text-halo-color": "#ffffff",
+      "text-halo-width": 1,
+    },
+  },
+  {
+    id: "places_region",
+    type: "symbol",
+    source: "protomaps",
+    "source-layer": "places",
+    filter: ["==", "pmap:kind", "region"],
+    layout: {
+      "symbol-sort-key": ["get", "pmap:min_zoom"],
+      "text-field": [
+        "step",
+        ["zoom"],
+        ["get", "name:short"],
+        6,
+        ["get", "name"],
+      ],
+      "text-font": ["Barlow Regular"],
+      "text-size": ["interpolate", ["linear"], ["zoom"], 3, 11, 7, 16],
+      "text-radial-offset": 0.2,
+      "text-anchor": "center",
+      "text-transform": "uppercase",
+    },
+    paint: {
+      "text-color": "#b3b3b3",
+      "text-halo-color": "#ffffff",
+      "text-halo-width": 2,
+    },
+  },
+  {
+    id: "places_country",
+    type: "symbol",
+    source: "protomaps",
+    "source-layer": "places",
+    filter: ["==", "pmap:kind", "country"],
+    layout: {
+      "symbol-sort-key": ["get", "pmap:min_zoom"],
+      "text-field": "{name}",
+      "text-font": ["Barlow Medium"],
+      "text-size": [
+        "interpolate",
+        ["linear"],
+        ["zoom"],
+        2,
+        [
+          "case",
+          ["<", ["get", "pmap:population_rank"], 10],
+          8,
+          [">=", ["get", "pmap:population_rank"], 10],
+          12,
+          0,
+        ],
+        6,
+        [
+          "case",
+          ["<", ["get", "pmap:population_rank"], 8],
+          10,
+          [">=", ["get", "pmap:population_rank"], 8],
+          18,
+          0,
+        ],
+        8,
+        [
+          "case",
+          ["<", ["get", "pmap:population_rank"], 7],
+          11,
+          [">=", ["get", "pmap:population_rank"], 7],
+          20,
+          0,
+        ],
+      ],
+      "icon-padding": [
+        "interpolate",
+        ["linear"],
+        ["zoom"],
+        0,
+        2,
+        14,
+        2,
+        16,
+        20,
+        17,
+        2,
+        22,
+        2,
+      ],
+      "text-transform": "uppercase",
+    },
+    paint: {
+      "text-color": "#b8b8b8",
+    },
+  },
+];
+
+export const BASEMAP_STYLE: StyleSpecification = {
+  version: 8,
+  sources: {
+    protomaps: {
+      type: "vector",
+      attribution:
+        '<a href="https://github.com/protomaps/basemaps">Protomaps</a> © <a href="https://openstreetmap.org">OpenStreetMap</a>',
+      url: `pmtiles://${process.env.NEXT_PUBLIC_S3_BUCKET_URL}/basemaps/20240325.pmtiles`,
+    },
+    counties: {
+      type: "vector",
+      url: `pmtiles://${process.env.NEXT_PUBLIC_S3_BUCKET_URL}/basemaps/tiger/tiger2023/tl_2023_us_county_full.pmtiles`,
```

Comment:
if this isn't passed in as a JSON, I think we'd want to pull out the file names as constants too instead of being hardcoded in

Reply:
Fair enough – for now if it's okay I'm going to leave it in since counties don't really change very often